### DISCUSSION TITLE
Neural_receiver.ipynb error

### DISCUSSION THREAD

**Discussion by kpasad** [NVlabs/sionna#191] [Q&A]
Running the virgin code gives the attached error.
![image](https://github.com/NVlabs/sionna/assets/14968621/4ced97ff-5177-4193-bd08-3f46c9f62d0d)

Any hints?

**Comment by SebastianCa** [comment]
Hi,

this could be related to #186. Could you please check your scipy version?

**Reply by kpasad** [reply]
Thanks. Issue is seen in Windows 7, 11 and Ubuntu. 
Rolling back scipy version did fix the problem.
scipy = 1.10.1

**Reply by kpasad** [reply]
In another manifestation of the same environment (replicated the environment on a different VM instance), setting the version to 1.10.1 does not solve the issue. One notable diffrence is 
typing_extensions=4.7.1=pyha770c72_0 (works)
typing_extensions=4.7.1=py311h06a4308_0 (does not work)

I could not figure out what the appendages to version mean.

**Comment by faycalaa** [comment]
Fixed by v0.15.1.
